tools/PI/DevHome.PI/Helpers/SnapHelper.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using CommunityToolkit.Mvvm.ComponentModel;
+using DevHome.Common.Extensions;
+using DevHome.PI.Models;
+using DevHome.PI.ViewModels;
+using Microsoft.UI.Xaml;
+using Windows.Graphics;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.Accessibility;
+using WinUIEx;
+
+namespace DevHome.PI.Helpers;
+
+public class SnapHelper
+{
+    // TODO The SnapOffsetHorizontal and UnsnapGap values don't allow for different DPIs.
```

Comment:
Does DevHome have an issue tracking this?

Reply:
Yes we have an issue created to fix all DPI handling in PI. Similar to #3031